pkg/features/features.go

```diff
@@ -146,11 +146,7 @@ func Get(names []string) ([]*Feature, error) {
 				return features, fmt.Errorf("Conflicting features: %s and %s", FeatureElementalRootfs, FeatureImmutableRootfs)
 			}
 
-			units := []*systemd.Unit{
-				systemd.NewUnit("elemental-rootfs.service"),
```

Comment:
good point I am not sure if this will still be de case. I mean the unit is there but mostly managed by the initramfs systemd unit.

The thing is that I to make `inst_simple` command in `module-setup.sh` functional having the service file within the module makes it easy. Probably we could use the full path in there as long as the unit file and the dracut module are in the same feature it shouldn't be a problem having a hardcoded full path in there. I can give it a quick test.